samples/skills/GroundingSkill/ReferenceCheckEntities/skprompt.txt

```diff
@@ -55,13 +55,18 @@ And a realio, trulio, little pet dragon
 Below are the reference items, and the context that you need to parse for references to the items.
 
 <input>
+<entities>
+{{$entities}}
+</entities>
+<context>
 {{$input}}
+</context>
 </input>
 
 <grounding_context>
 {{$reference_context}}
 </grounding_context>
 
-Please put your response after the "Response:". Produce a bulleted list of the <ungrounded_entities> which are not grounded in the context, and then repeat the <context>
+Please put your response after the "Response:". Produce a bulleted list of the <ungrounded_entities> which are not grounded in the context.
```

Comment:
Given this change, does the example need updating above?

Reply:
(Sorry, GitHub won't let me highlight the lines directly.... L46-51

Reply:
Good catch!